test/avocado/selenium-machines-basic.py

```diff
@@ -107,3 +108,43 @@ def testDelete(self):
         self.assertNotIn(name, self.machine.execute("sudo virsh list --all"))
         self.assertNotIn(imgdel, self.machine.execute("sudo virsh vol-list {}".format(args.get('poolName'))))
         self.assertIn(args.get('image'), self.machine.execute("sudo virsh vol-list {}".format(args.get('poolName'))))
+
+    def testVmStatus(self):
+        name = 'staticvm'
+        self.create_vm(name)
+
+        self.assertEqual(
+            self.machine.execute('sudo virsh domstate {}'.format(name)).rstrip(),
+            self.wait_css('#vm-{}-state'.format(name)).text)
+
+    def testCreate20VMs(self):
+        iso_source = '/home/{}.iso'.format('test' + str(time.time()).split('.')[0])
```

Comment:
For random strings you can use https://docs.python.org/2/library/random.html instead.
I don't insist, you can keep this one as is.